aptos-move/framework/aptos-framework/sources/randomness.spec.move

```diff
@@ -3,7 +3,9 @@ spec aptos_framework::randomness {
     spec module {
         use aptos_framework::chain_status;
         pragma verify = true;
-        invariant [suspendable] chain_status::is_operating() ==> exists<PerBlockRandomness>(@aptos_framework);
+        invariant[suspendable] chain_status::is_operating() ==> exists<PerBlockRandomness>(
```

Comment:
There should be a space between `invariant` and `[suspendable]`

Reply:
we will fix it

Reply:
fixed at https://github.com/movebit/movefmt/commit/d94da4a5e323435fec82a90b4ae06c6d5ed5a0a2